docs/dialect.md

```diff
@@ -173,3 +173,110 @@ Allowable date parts: year, quarter, month, dayofyear, day, weekday, week, hou
+  Literals is used when a boolean is used as a value. Default to: `x ? 1 : 0` <br/>
+  Example usage: `SELECT 1 as literalBool` <br/>
+  Predicates is used when a boolean is used as a predicate. Default to: `isTrue ? (1=1) : (1=0)` <br/>
+  Example usage: `SELECT something as test WHERE (1=1)` <br/>
+  `value` attribute is used as `predicate` for backwards compatability <br/>
+  Example:
+  ```xml 
+    <format-true literal='TRUE' predicate='TRUE' />
+  ```
+  ```xml 
+    <format-false literal='FALSE' predicate='FALSE' />
+  ```
+
+   **format-bool-as-value** <br/>
+    Used in CASE statements. Determines whether the true or false case is used first. The fucntion is only used when `CAP_QUERY_BOOLEXPR_TO_INTEXPR` capiblity is set to yes. 
+    The value for this property can be: 
+  - __TrueFirst__
+      This is the default case. Logic: `CASE WHEN %1 THEN 1 WHEN NOT %1 THEN 0 ELSE NULL END`
+  - __FalseFirst__
+      False case is used first. Logic: `CASE WHEN NOT %1 THEN 0 WHEN %1 THEN 1 ELSE NULL END`
+
+  ```xml 
+    <format-bool-as-value value='TrueFirst' />
+  ```
+  
+ **Boolean Capabilities**  <br/>
+  
+  Boolean usage is also defined by capabilities in the manifest file. <br\>
+  See the Query section for boolean capabilities [here]({{ site.baseurl }}/docs/capabilities) for more details.
+
+ **TDVT Coverage**  <br/>
+The following TDVT tests check that the boolean functionality is working as expected for a connector. 
+logical.bool	`exprtests\standard\setup.logical.bool.txt` and logical	`exprtests\standard\setup.logical.txt`. See [this](https://github.com/tableau/connector-plugin-sdk/tree/master/tdvt/tdvt/exprtests/standard) for more details. 
+
+### Temporary Table Support:
+   **format-create-table** <br/> 
+  This function uses  piece-by-piece formulas for creating a table.
+  Predicates can be used with tokens that only corresponds to a certain type of table.
+  Predicates: <br/>
+  -  __GlobalTemp__
+  -  __LocalTemp__
+  -  __AnyTemp__
+  -  __NoTemp__ <br/>
+   You should also be able to use the following string substitution tokens along with the predicate:
+      - __%n - table name__
+     - __%f - formatted column list__
+    This function is avialable when `CAP_CREATE_TEMP_TABLES` capability is set to yes. 
+  
+  ```xml 
+    <format-create-table>
+        <formula>CREATE </formula>
+        <formula predicate='GlobalTemp'>GLOBAL </formula>
+        <formula predicate='LocalTemp'>LOCAL </formula>
+        <formula predicate='AnyTemp'>TEMPORARY </formula>
+        <formula>TABLE %n (</formula>
+        <formula>%f</formula>
+        <formula>)</formula>
+        <formula predicate='AnyTemp'> ON COMMIT PRESERVE ROWS</formula>
+      </format-create-table>
+  ```
+
+  **format-select** <br/>
+  This function uses piece-by-piece formula for defining a SELECT statement. Here, we can define the clause used in `SELECT` statement. <br/>
```

Comment:
Grammar nit: "This function uses _a_ piece-by-piece formula..."
"Here, we can define the _clauses_ used in _the_ `SELECT` statement."